frontend/src/pages/UserItineraries.js

```diff
@@ -1,31 +1,50 @@
 import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import itineraryAPI from '../api/itineraryAPI.js';
+import boardAPI from '../api/boardAPI.js';
 import '../styles/styles.css';
 
 const username = localStorage.getItem('username');
 
 function Itineraries() {
-  const [itineraries, setItineraries] = useState([]);
+  const [itineraryEvents, setItineraryEvents] = useState([]); // 2D array for itineraries and events
```

Comment:
I'm getting an error because of this change. setItineraries is still being called.

Reply:
Apologies, I have fixed this